pkg/proc/bininfo.go

```diff
@@ -59,6 +59,8 @@ type BinaryInfo struct {
 
 	loadErrMu sync.Mutex
 	loadErr   error
+
+	aordr *dwarf.Reader // extra reader to load DW_AT_abstract_origin entries
```

Comment:
Maybe call this `dwarfReader` instead of naming it specifically for abstract origin usage, even though that's the only place we're using it now, we could possibly use this reader other places and seek instead of allocating.